content/en/data-playground/tabbed-test.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Should the test file be removed before merging or should `draft` be changed to `false`?